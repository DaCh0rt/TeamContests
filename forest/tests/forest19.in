575 470
9965 -3116
7480 -6108
53 -5188
3097 -1172
6401 -7652
-9221 4281
-1772 -6284
-9396 7942
-4460 -2171
8037 -1696
-9557 -6969
-4374 -4530
-3713 3824
9975 9637
3620 -604
-4332 -9148
-1596 -4487
-3117 -2519
3853 -5246
-1234 8849
-1970 -1495
5372 7413
-6543 2817
-5138 2816
9367 8658
2555 6922
9967 -8516
-7896 -3435
6793 4984
-882 6296
9769 -9930
-5067 7456
2851 -6223
-7518 -863
992 5295
-6583 5999
4185 -6630
1901 1912
-6987 -9844
-9049 9842
-9179 1656
4259 -3136
5698 7054
-1209 2242
-9748 -1724
-817 1657
-310 -7609
5963 386
1212 -13
-6222 -9587
8661 -3279
-2491 -1430
9145 2960
4729 2276
-7943 -6793
9510 -6305
7064 4557
-9656 -7081
-7901 -6148
-2021 15
9088 5284
2931 9846
9354 -2095
-8636 -1846
5657 -5506
7806 -8563
-2081 3292
1418 -3807
8468 -9982
-5003 -743
9332 8747
8081 -6761
400 -4498
-6737 -4224
3080 -9363
1839 8618
6562 -21
9770 522
-1635 3105
-1969 4832
2225 -2403
7152 7068
8978 -7875
-3414 8324
6896 9491
141 -3660
4744 9574
-6301 7588
-1365 -8706
-8158 3088
-2169 9923
7198 2643
1172 166
-2913 -3030
52 -2626
-9791 -1559
3134 -6384
2598 -4370
5857 -6535
5590 374
962 8109
8275 2387
408 -4439
-8841 1257
-582 -192
703 -3827
6002 1929
5525 6146
4805 7762
-9450 3370
-4779 -5261
-9986 8411
2903 -6426
-8272 2019
-9378 9125
-4752 3197
4410 7123
698 4531
9599 -9816
4119 -142
-1592 9414
-5365 -6786
8880 -8906
-3230 -8635
-2412 -3995
5177 1014
311 9269
3031 -4086
-5431 1848
623 790
6781 9132
-7367 7355
-1454 -1598
-4330 1624
-3204 -3624
-3191 5070
8101 9439
7622 277
-2779 -4684
-2514 7837
-6564 5625
-9969 -6039
-367 4067
-6462 -2148
437 5790
-1721 6709
547 -6420
-7833 3707
-7286 -2593
2178 -8381
4848 8825
-9120 4900
-960 991
-4339 4478
-7370 3051
-6345 9070
-1216 -3023
6144 -1005
-9372 6928
9230 -591
-6500 8044
9646 -4504
6929 -6125
-5088 6994
7000 -1950
3907 5843
-7862 -2490
3875 -1367
-5218 9701
-8378 3022
-3581 -8569
7396 9974
-3307 -8531
2613 5604
-5956 -9577
-2593 -343
7059 -5634
-9273 -1628
-1922 9623
5895 -7028
9261 -8098
4223 1813
-9401 -155
2526 -5103
7415 5397
-833 -6791
-2175 63
6544 -3901
3340 -4548
6048 3559
-3159 5348
3388 2063
7972 -9542
6867 -1297
3306 3784
7963 4694
2753 -1621
-2679 -7608
2156 3231
-9352 -9616
-9029 -8028
8724 -713
7409 -1804
-3622 3605
6005 3540
-7542 5234
-7901 -777
-7118 -7174
3883 487
-342 -9815
-5444 -7235
-1836 -6092
-6246 7614
2962 -30
-156 -5041
6175 -337
7661 -1780
9711 8085
-3419 -4080
2371 9168
4229 -7294
-2552 5898
-4542 4007
9091 5982
-1736 3256
8161 -9723
-3421 4554
-5501 1644
-3088 5550
1115 3039
2229 -5818
19 -5323
6087 6912
6583 -7725
-9035 2084
-8524 -9271
1571 -6029
-771 -6807
-5673 8935
9928 -9912
6358 2384
569 -2897
2522 -4685
-5863 -6567
7576 6480
-7132 -8913
2774 -9548
-4172 5509
4945 8973
3771 -49
-3502 -319
-9343 -7530
-6788 9209
-6065 -8597
-2899 7347
2904 -7864
-4991 1209
-4279 -5382
9858 -1639
-3427 1386
-4374 -7968
-3586 6066
6811 2456
-9556 -4471
1237 3863
3546 5040
-6547 2555
-3873 9930
4437 -1125
-8224 -2496
-8220 -8245
9382 -8847
2011 -6545
-1678 -8190
2625 5851
5438 7673
7430 5465
2197 -5810
-9921 -5306
8581 6366
-9001 -4432
-1857 9966
-6743 -3584
-6588 -7685
2606 -779
9818 -4442
-9993 5235
-8263 -1791
-9093 -856
-4333 9541
-3357 -3715
6677 6753
-1978 -7850
-7400 -2440
-1178 6290
9921 7061
6586 -7727
6056 -9154
-2557 1202
-5479 578
-517 4757
-4690 2015
-3527 -5652
-2716 4563
-2482 -4876
5086 -4073
1237 3271
-7232 -9887
-1141 8401
7442 407
2625 -5726
8985 8333
-4413 7777
-9749 -4752
2188 -5643
7218 -974
3002 -7668
9534 -8581
-5014 -9137
-2650 -1771
-8808 -2777
-5399 9361
6827 8305
-5286 -5914
-6381 1202
7299 -6084
-8388 5401
-9319 -2164
1309 -5074
-7487 -9966
1139 365
622 7069
3667 -8104
-3846 4502
6022 5220
3652 4992
1667 3816
-4284 3248
-9710 7065
1423 7637
3547 762
-5117 -6561
4388 4100
96 3435
134 -2706
-6770 -6322
-4486 -3790
-6242 2102
5685 -280
-1649 -7364
-9350 -7068
-5930 860
-6437 3939
3398 -806
-9491 -1460
1708 4911
8929 -7436
4105 -4678
-7088 2064
-8589 -6366
3311 188
-4108 2003
-7293 -6733
-6756 -5307
-90 9418
6643 2765
9452 7406
-3530 6450
5081 -4927
3782 7142
-6263 6176
7394 -5401
-8296 -3932
7457 1761
-9754 -2279
2408 5445
10000 -8572
-7901 304
590 -6169
9525 9880
8419 1651
3890 -5104
88 -9603
-7026 -1722
4942 -6703
-3611 -3468
-7273 4481
4889 1884
-3225 -2205
5933 -752
-5932 1271
-8415 3096
8880 -4317
-4407 7994
-9272 -7973
-5630 -3486
4588 -4163
-4566 2956
-7175 3943
-8349 9775
-4554 1004
3673 8451
-7466 774
5594 236
8865 9996
8814 -732
-2706 5081
6435 7734
5174 -9774
-1647 9580
-2861 363
4725 4333
1148 539
-9542 4884
-9233 -7677
8417 1908
8614 -8380
-5579 -8062
-1221 -5427
-2712 8079
-3534 21
7338 -1449
2473 -9224
-2823 1414
5003 -595
-7726 6959
6548 -3391
8263 -9537
5941 8150
-3694 -1460
1165 4694
-4746 9614
-167 9948
-529 1779
8479 5826
991 5157
2690 8804
9919 2866
-9154 1602
620 9159
-737 1466
-9125 -454
-6054 8021
1617 -9106
-6687 -3858
5817 4789
-3676 2821
-2287 -5841
775 943
4655 1025
7889 -37
-9591 -5971
-2685 6087
-3367 -1546
-2264 -4323
-1834 -7986
-6069 9837
-2390 -5351
9194 -5631
2563 231
8407 -1776
3325 4900
8564 -6129
-1875 2084
-4200 1432
2547 7941
-2729 -1288
3049 -8745
-7355 -4527
5788 -9524
-3347 3504
4466 1326
609 1220
8490 -5588
-6833 -4807
-942 -7582
6819 7713
-9704 6785
-4994 1717
3536 -4883
7235 -5890
-2408 518
6295 1287
4931 9912
-1599 -1637
-7780 -1070
-6765 -4831
-9142 -271
-1705 -6979
-3710 3026
366 -3879
-5476 -2298
-1149 -2381
-8136 -5019
5686 837
3250 9536
-2010 9211
-1275 -1729
-2121 41
7122 -5841
3308 -6256
-5337 -1714
-6123 60
-8737 3725
1542 3814
-2117 7571
-2992 1844
2771 -854
-7999 -8005
-640 -1219
-4462 -7627
-3106 9935
5921 3517
5438 959
-3649 -2076
2768 -6023
-9527 5269
9948 5859
6986 4952
-6557 -1501
2755 7854
1357 6393
2458 139
-7214 688
6462 -8781
7392 -3334
1998 -2478
-4949 9162
-9999 496
-607 4269
194 -1770
7728 -7438
9376 -4155
-4656 4448
-7011 -8886
4681 4987
9439 -7673
3640 8914
2647 -4252
4761 -5731
287 8311
-5134 -3477
5519 -3381
-323 -2843
-5424 -1555
333 3879
8945 -7811
8025 9740
-4463 4380
4302 -7904
5509 5518
-9171 6158
7825 6207
-7772 2819
-4822 2489
1116 6916
-9743 -851
5406 -8139
-4545 -8983
4385 -3402
3196 8940
2139 9111
4969 -6601
7824 -311
-1124 -3969
1897 1107
7207 5645
5862 -9067
5970 -911
-3030 -5335
9538 7478
-763 8909
-339 5939
1396 1675
-1464 -4342
