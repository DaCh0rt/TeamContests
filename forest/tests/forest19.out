80705.69 432567466.61
